src/system.h

```diff
@@ -181,6 +181,8 @@ struct seccomp_notif_resp {
 						     struct seccomp_notif_resp)
 #define SECCOMP_IOCTL_NOTIF_ID_VALID    SECCOMP_IOR(2, __u64)
```

Comment:
My apologies @sroracle, I forgot about this line.  Yeah, we should probably update `SECCOMP_IOCTL_NOTIF_ID_VALID` to match the kernel.  You're correct.